frontend/components/Editor.js

```diff
@@ -1117,6 +1106,20 @@ patch: ${JSON.stringify(
         })
     }
 
+    componentDidMount() {
+        if (this.state.static_preview) {
+            console.log("setting state to initializing: false")
+            this.setState({
+                initializing: false,
+                binder_phase: this.state.offer_binder ? BinderPhase.wait_for_user : null,
+            })
```

Comment:
Setting the state in the constructor was never actually going through, so instead we wait for the component to mount, _then_ set initializing to false to trigger the static render logic.